src/server.c

```diff
@@ -2621,6 +2621,11 @@ void initServer(void) {
     server.client_mem_usage_buckets = NULL;
     resetReplicationBuffer();
 
+    if (server.maxmemory) {
+        server.maxmemory_available =
+            (unsigned long long)server.maxmemory / 100.0 * (100 - server.maxmemory_reserved_scale);
```

Comment:
extract this formula to an inline function? this has been repeated 3 times

Reply:
Address. I just create an inline function in server.h